### PR TITLE
About: Quote Section

### PR DESCRIPTION
**Pull request by thrlstl**
https://trello.com/c/43zWHo6Z/102-about-quote-section

- I believe I was able to reverse all of the formatting changes.
- Font-weight issue resolved by adding 400 weight in initial google font import in main.scss.
- This should be ready for review now.

### CODE REVIEW

File: pages/about.html [hunk 1]
```diff
@@ -5,4 +5,19 @@
...
\ No newline at end of file
+<div class='quote-content-container'>
+  <div class='civic-info-container'>
+    <h1>What is Civic Tech?</h1>
+    <p>Civic tech bridges the gap between the civic space and technology. It’s utilizing technology practices like
+      design, engineering, data science, etc. to optimize the processes for civic organizations.</p>
+  </div>
+  <div class='quote-content'>
+    <div class='quote-paragraph-container'>
+      <p1>“...those of us who work in civic tech want public digital goods to be as good as the ones made by commercial
```
Comment (jrapala) [inline pages/about.html:16]: Lets use standard HTML tags, like `<p>`

Or better yet a [blockquote](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote)

File: pages/about.html [hunk 2]
```diff
@@ -5,4 +5,19 @@
...
+        entities like Apple or Google—and we want public digital infrastructure to be as good, too.” </p1>
+      <br>
+      <p2>Cyd Harrell, A Civic Technologist’s Practice Guide</p2>
+    </div>
+    <img class="quote-icon" src="/images/about-page/quote.svg" alt="Quote Icon">
```
Comment (jrapala) [inline pages/about.html:21]: Nice use of the alt attribute! However, this icon is entirely decorative. Can we set `alt=""`?

File: pages/about.html [hunk 3]
```diff
@@ -5,4 +5,19 @@
 home: false
 ---
 
-<h2>About Content Goes Here</h2>
\ No newline at end of file
+<div class='quote-content-container'>
+  <div class='civic-info-container'>
+    <h1>What is Civic Tech?</h1>
```
Comment (jrapala) [inline pages/about.html:10]: There are a number of these section on about.html
Let's use an H2 instead

File: scss/main.scss [hunk 4]
```diff
@@ -1119,4 +1119,129 @@ p.cardText {
...
+  }
+  @media (max-width: 651px) {
+    h1 {
+      font-size: 32px;
+      width: 555px;
+    }
+    p {
+      font-size: 18px;
+      line-height: 23px;
+      width: 555px;
```
Comment (jrapala) [inline scss/main.scss:1174]: Between 360px and 651px, this paragraph is 555px, so it goes off the screen. Can we please fix?

**Review by jrapala — CHANGES_REQUESTED** [review]
Nice job! I left a few comments in the PR.
Also, here are a few additional comments:
- Can every font-size property use rem units instead of pixels?
- There are a few hardcoded hex values like #333333 that should be replaced with variable names. Variables exist in _variables.scss
- The quote-content-container styles are used all over the about page. Perhaps we can make the class name a little more generic?

File: scss/main.scss (hunk 4 above)
Comment (thrlstl) [reply scss/main.scss]: I don't see this issue on my end, perhaps it's resolved. Let me know if you're still seeing this.

**Comment by thrlstl** [comment]
- [ ] Can every font-size property use rem units instead of pixels?
- Let's touch base on this, I'm not sure what our base is coming from and am unable to calculate the proper rem units per Joseph's designs.
- [x] There are a few hardcoded hex values like #333333 that should be replaced with variable names. Variables exist in _variables.scss
- [x] The quote-content-container styles are used all over the about page. Perhaps we can make the class name a little more generic?

**Comment by jrapala** [comment]
Use 16px as the base font size.
Can you fix the merge conflict please?

**Comment by jrapala** [comment]
I see that you are using Node 15 or above. Can you please downgrade to something less than 15 (e.g. 14) and reinstall your dependencies?
I updated the README with a note about this. Sorry for not warning earlier.

File: scss/main.scss [hunk 5]
```diff
@@ -1141,6 +1141,151 @@ p.cardText {
...
+  }
+
+  @media (max-width: 651px) {
+    .content-header{
+      h2 {
+        font-size: 2rem;
+        width: 555px;
+      }
+      p {
+        font-size: 1.125;
```
Comment (jrapala) [inline scss/main.scss:1199]: This needs a unit

File: scss/main.scss [hunk 6]
```diff
@@ -1141,6 +1141,151 @@ p.cardText {
...
+        line-height: 23px;
+        width: 555px;
```
Comment (jrapala) [inline scss/main.scss:1201]: This section extends past the width of the screen on sizes less than 555px.

File: scss/main.scss [hunk 7]
```diff
@@ -1141,6 +1141,151 @@ p.cardText {
...
+      font-style: normal;
+      font-weight: normal;
+      font-size: 1.25rem;
+      line-height: 30px;
+      color: $darker-grey;
+    }
+  }
+  
+  @media (max-width: 768px) {
+    padding: 64px 48px;
```
Comment (jrapala) [inline scss/main.scss:1181]: The horizontal margin on smaller screens should be 24px

Screen Shot 2021-05-04 at 7 39 18 PM

File: scss/main.scss [hunk 8]
```diff
@@ -1141,6 +1141,151 @@ p.cardText {
...
+ }
+  
+ .quote-content {
+  position: relative;
+  max-width: 1440px;
+  .quote-paragraph-container {
+    padding-top: 80px;
+    padding-left: 107px;
+    height: max-content;
+    transition: ease all .5s;
```
Comment (jrapala) [inline scss/main.scss:1229]: No need for these transition properties. Can we remove all of them, please? (Animating things like width and padding are very expensive).

**Review by jrapala — CHANGES_REQUESTED** [review]
Left a few more comments. Can you please take a look at the Node issue as well?

Thanks!

File: pages/about.html [hunk 9]
```diff
@@ -5,4 +5,20 @@
...
+        <h2>What is Civic Tech?</h2>
+        <p>Civic tech bridges the gap between the civic space and technology. It’s utilizing technology practices like
+            design, engineering, data science, etc. to optimize the processes for civic organizations.</p>
+    </div>
+    <div class='quote-content'>
+        <div class='quote-paragraph-container'>
+            <p class='quote-paragraph'>“...those of us who work in civic tech want public digital goods to be as good as
+                the ones made by
+                commercial
+                entities like Apple or Google—and we want public digital infrastructure to be as good, too.” </p>
```
Comment (jrapala) [inline pages/about.html:19]: Small thing--Can you clean up the odd spacing in this quote? Lots of line breaks in the code

**Review by jrapala — DISMISSED** [review]

**Review by jrapala — CHANGES_REQUESTED** [review]
Did you switch to npm 6?
If so, please commit your new package-lock.json file. Thanks!

**Comment by thrlstl** [comment]
> Did you switch to npm 6?
> If so, please commit your new package-lock.json file. Thanks!

Should be good now.

**Review by jrapala — APPROVED** [review]

**Pull request merged**